sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -302,6 +329,15 @@ def __repr__(self):
             :1024
         ]
 
+    def to_dict(self):
+        return {
+            "value_type": self.value_type,
+            "name": self.name,
+            "value": self.value,
```

Comment:
A FormField.value can return a Dict[str, FormField] or a List[FormField] as well (e.g. the "Items" field in Invoices). I think we'll need some logic here to check for that and make sure those FormFields are getting converted to dicts.

Reply:
Done

Reply:
Let me know if the if check is good in this case